htsinfer/infer_count_motifs.py

```diff
@@ -0,0 +1,91 @@
+"""Infer motif frequency from sample data."""
+
+
+from typing import Dict
+
+
+def count_motifs(input_sequences: list, min_motif_length: int,
+                 max_motif_length: int, nucleic_acid: str) -> dict:
+    """Function that calculates the occurrence of all motifs in one or
```

Comment:
Best to keep these short, precise and free of redundancy. The function signature and context make it clear that this is a function. And the `Returns` section specifies in more detail what is returned.

Better would be something like: "Count motifs on one or more nucleotide sequences"